#! /bin/sh /usr/share/dpatch/dpatch-run
## 32_check_ldap_pointer.dpatch by Jan Wagner <waja@cyconet.org>
## patch provided by dann frazier <dannf@debian.org>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Using deprecated interfaces of libldap API

@DPATCH@
--- nagios-plugins-1.4.11.orig/plugins/check_ldap.c	2007-07-07 16:20:40.000000000 -0600
+++ nagios-plugins-1.4.11/plugins/check_ldap.c	2008-01-30 15:14:14.000000000 -0700
@@ -42,6 +42,7 @@
 #include "utils.h"
 
 #include <lber.h>
+#define LDAP_DEPRECATED 1
 #include <ldap.h>
 
 enum {
